Packs/Phishing/Scripts/PhishingDedupPreprocessingRule/PhishingDedupPreprocessingRule.py

```diff
@@ -247,10 +248,20 @@ def create_new_incident():
     return_entry('This incident is not a duplicate of an existing incident.')
 
 
-def create_new_incident_low_similarity(existing_incident, similarity):
-    message = 'This incident is not a duplicate of an existing incident.\n'
-    message += 'Most similar incident found is #{} with similarity of {:.1f}%.\n'.format(existing_incident['id'],
-                                                                                         similarity * 100)
+def format_similar_incident(incident, similairy):
+    return {'Id': "[%s](#/Details/%s)" % (incident['id'], incident['id']),
+            'Name': incident['name'],
+            'Closed': incident.get('closed') != "0001-01-01T00:00:00Z",
+            'Time': str(incident['created']),
+            'Email from': incident.get(demisto.args().get('emailFrom')),
+            'Text Similarity': "{:.1f}%".format(similairy * 100),
+            }
+
+
+def create_new_incident_low_similarity(existing_incident, new_incident, similarity):
+    message = '# This incident is not a duplicate of an existing incident.\n'
+    formatted_incident = format_similar_incident(existing_incident, similarity)
+    message += tableToMarkdown("Most similar incident found", formatted_incident)
```

Comment:
perhaps we should add a minimum threshold of displaying the original message, or even 2 different messages for a certain threshold
meaning:  if x<similarity<0.99 - show most similar message
if similarity < x :  we can just say that no duplicated incident was found
for x=0.8 or x=0.85
what do you think?

Reply:
agree